scripts/autopep8.sh

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env bash
+
+###############################################################################
+# Copyright 2020 The Apollo Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###############################################################################
+
+# Usage:
+#   autopep8.sh <path/to/src/dir/or/files>
+
+# Fail on error
+set -e
+
+TOP_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd -P)"
+source "${TOP_DIR}/scripts/apollo.bashrc"
+
+AUTOPEP8_CMD="autopep8"
+
+function _find_py_srcs() {
+  find "$@" -type f -name "*.py"
+}
+
+function _py_ext() {
+  local __ext
+  __ext="$(file_ext $1)"
+  if [ "${__ext}" == "py" ]; then
+    return 0
+  fi
+  return 1
+}
+
+function check_autopep8() {
+  AUTOPEP8_CMD="$(command -v autopep8)"
+  if [ -z "${AUTOPEP8_CMD}" ]; then
+    error "Oops, autopep8 missing..."
+    error "Please make sure autopep8 is installed and check your PATH" \
+        "settings. For Debian/Ubuntu, you can run the following command:"
+    error "  sudo pip install --upgrade --no-cache-dir autopep8"
+    exit 1
+  fi
+}
+
+function autopep8_run() {
+  ${AUTOPEP8_CMD} -i -a -a "$@"
```

Comment:
Two `-a` ?

Reply:
it means the aggressive level, multiple -a result in more aggressive changes.